.yarnrc

```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+yarn-path ".yarn/releases/yarn-1.22.4.js"
```

Comment:
This is the result of `yarn policy set-version`: it guarantees that everybody will use the same `yarn` version when running commands inside this repo. That file points to a specific `yarn` release, it is downloaded by `yarn` itself.